R/utilities.R

```diff
@@ -180,3 +233,16 @@
     )
     stop(msg, call. = FALSE)
 }
+
+.dummy_iter <- function(X){
```

Comment:
This seems like it's used in only one location, and could be moved and renamed `.bploop_lapply_iter`

Reply:
Thanks, I'll rename it.